src/components/craft/inputs/SizeStyleInput.tsx

```diff
@@ -8,30 +8,56 @@ export const extractNumber = (string?: string) =>
 
 export const extractSizeUnit = (string?: string) => string?.match(/px|%|rem|em/g)?.[0]
 
-export type SizeStyle = Pick<CSSObject, 'width' | 'height'>
+export type SizeStyle = Pick<CSSObject, 'width' | 'height' | 'backgroundImage'>
+type imgProps = {
+  width: number
+  height: number
+  aspectRatio: number
+  gcd: number
+  widthAspect: number
+  heightAspect: number
+}
 type SizeStyleInputProps = {
   value?: SizeStyle
+  imgProps: imgProps
+  isImgAutoHeight: boolean
   onChange?: (value: SizeStyle) => void
 }
-const SizeStyleInput: React.VFC<SizeStyleInputProps> = ({ value, onChange }) => {
+
+const SizeStyleInput: React.VFC<SizeStyleInputProps> = ({ value, imgProps, isImgAutoHeight, onChange }) => {
   const { formatMessage } = useIntl()
 
   return (
     <>
       <Form.Item label={formatMessage(craftMessages['*'].width)}>
         <InputNumber
-          value={extractNumber(value?.width?.toString())}
+          value={value?.width !== undefined ? extractNumber(value?.width?.toString()) : imgProps.width}
```

Comment:
NITS: avoid negative expression is more readable
```ts
value?.width === undefined ? imgProps.width : extractNumber(value?.width?.toString()) 
```

Reply:
had modified in below commit.
thanks.